service-api/src/test/java/greencity/validator/BooleanValueDeserializerTest.java

```diff
@@ -0,0 +1,28 @@
+package greencity.validator;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import greencity.exception.exceptions.NotValidBooleanValueException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BooleanValueDeserializerTest {
+
+    private final BooleanValueDeserializer deserializer = new BooleanValueDeserializer();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    @Test
```

Comment:
You can combine 2 tests into 1 by ParameterizedTest:
```
    @ParameterizedTest
    @ValueSource(strings = {"invalid value", ""})
    void deserializeInvalidValueTest(String jsonValue) throws IOException {
        JsonParser jsonParser = objectMapper.getFactory().createParser(jsonValue);
        assertThrows(NotValidBooleanValueException.class, () -> deserializer.deserialize(jsonParser, null));
    }
```

Reply:
done